owl/EASE-OBJ.owl

```diff
@@ -526,10 +779,55 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Clean -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Clean">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#CleanlinessRegion"/>
+        <rdfs:comment>A cleanliness region with values considered as clean.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Cleanliness -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Cleanliness">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#SocialQuality"/>
```

Comment:
this is maybe a bit questionable, my rationale was that this quality is defined within a social context.

Reply:
Actually that is probably exactly right.

One can objectively measure the number of contaminants etc, but one has to make a commitment that this is a contaminant whereas this other thing is ok to have. Such a decision is definitely the creation of an agent.